Add RSS page (#895)
privacyguides/privacyguides.org: docs/news-aggregators.md

---
title: News Aggregators
icon: octicons/rss-24
---

> [News aggregators](https://en.wikipedia.org/wiki/News_aggregator) are a a great way to keep an eye on your favourite blogs and places of news.

Author: it is a bit, I'm sure i can come up with something better.

!!! tip "Subscribe to YouTube channels without association to your Google Account"
    You can subscribe YouTube channels without logging in and associating usage information with your Google Account. To subscribe to a YouTube channel with an RSS client, first look for your [channel code](https://support.google.com/youtube/answer/6180214) and then add it to this URL: https://www.youtube.com/feeds/videos.xml?channel_id=`<channel id>`

!!! tip "Reddit aggregation support"
    Reddit also supports subscription via RSS, just add the subreddit name this URL: https://www.reddit.com/r/`<subreddit name>`/new/.rss.

### Fluent Reader

!!! recommendation

    ![Fluent Reader logo](/assets/img/news-aggregators/fluent-reader.svg){ align=right }

    **Fluent Reader** is a secure cross-platform news aggregator that has useful privacy features such as deletion of cookies on exit, strict [content security policies (CSP)](https://en.wikipedia.org/wiki/Content_Security_Policy) and proxy support, meaning you can use it over [Tor](/self-contained-networks/#tor).

>     [Visit hyliu.me](https://hyliu.me/fluent-reader){ .md-button .md-button--primary } [Privacy Policy](){ .md-button }

Reviewer: Missing privacy policy

```suggestion
    [Visit hyliu.me](https://hyliu.me/fluent-reader){ .md-button .md-button--primary } [Privacy Policy](https://github.com/yang991178/fluent-reader/wiki/Privacy){ .md-button }
```

    **Downloads**
    - [:fontawesome-brands-windows: Windows](https://hyliu.me/fluent-reader)

>     - [:fontawesome-brands-apple: macos](https://apps.apple.com/app/id1520907427)

Reviewer: Capitalization

```suggestion
    - [:fontawesome-brands-apple: macOS](https://apps.apple.com/app/id1520907427)
```

### Akregator

!!! recommendation

    ![Akregator logo](/assets/img/news-aggregators/akregator.svg){ align=right }

    **Akregator** is a news feed reader that is a part of the [KDE](https://kde.org) project. It comes with a fast search, advanced archiving functionality and an internal browser for easy news reading.

    [Visit kde.org](https://apps.kde.org/akregator){ .md-button .md-button--primary } [Privacy Policy](https://kde.org/privacypolicy-apps){ .md-button }

    **Downloads**
    - [:pg-flathub: Flatpak](https://flathub.org/apps/details/org.kde.akregator)
    - [:fontawesome-brands-git: Source](https://invent.kde.org/pim/akregator)

### Handy News Reader

!!! recommendation

    ![Handy News Reader logo](/assets/img/news-aggregators/handy-news-reader.svg){ align=right }

    **Handy News Reader** is a fork Flym that has many [features](https://github.com/yanus171/Handy-News-Reader#features) and works well with folders of RSS feeds. It supports [RSS](https://en.wikipedia.org/wiki/RSS), [Atom](https://en.wikipedia.org/wiki/Atom_(Web_standard)) and [RDF](https://en.wikipedia.org/wiki/RDF%2FXML).

>     [Visit github.com](https://github.com/yanus171/Handy-News-Reader){ .md-button .md-button--primary }

Reviewer: Use the GH pages website instead of the repo

```suggestion
    [Visit github.com](http://yanus171.github.io/Handy-News-Reader/){ .md-button .md-button--primary }
```

    **Downloads**
    - [:fontawesome-brands-google-play: Google Play](https://play.google.com/store/apps/details?id=ru.yanus171.feedexfork)
    - [:pg-f-droid: F-Droid](https://f-droid.org/en/packages/ru.yanus171.feedexfork/)
    - [:fontawesome-brands-github: Source](https://github.com/yanus171/Handy-News-Reader)

### NetNewsWire

!!! recommendation

    ![NetNewsWire logo](/assets/img/news-aggregators/netnewswire.png){ align=right }

>     **NetNewsWire** a free and open-source feed reader for macOS and iOS. It supports [RSS](https://en.wikipedia.org/wiki/RSS), [Atom](https://en.wikipedia.org/wiki/Atom_(Web_standard)), [JSON Feed](https://en.wikipedia.org/wiki/JSON_Feed), and [RSS-in-JSON](https://github.com/scripting/Scripting-News/blob/master/rss-in-json/README.md) formats.

Reviewer: This is the only recommendation that doesn't describe what sets it apart from other reader apps. 

```suggestion
    **NetNewsWire** a free and open-source feed reader for macOS and iOS with a focus on a native design and feature set. It supports the typical feed formats alongside built-in support for Twitter and Reddit feeds.
```

>     [Visit netnewswire.com](https://netnewswire.com/){ .md-button .md-button--primary } [Privacy Policy](){ .md-button }

Reviewer: Missing privacy policy

```suggestion
    [Visit netnewswire.com](https://netnewswire.com/){ .md-button .md-button--primary } [Privacy Policy](https://netnewswire.com/privacypolicy){ .md-button }
```

    **Downloads**

>     - [:fontawesome-brands-apple: macos](https://apps.apple.com/us/app/netnewswire-rss-reader/id1480640210)

Reviewer: Wrong link

```suggestion
    - [:fontawesome-brands-apple: macOS](https://netnewswire.com)
```

>     - [:fontawesome-brands-app-store: Mac App Store](https://apps.apple.com/us/app/netnewswire-rss-reader/id1480640210)

Reviewer: iOS not macOS

```suggestion
    - [:fontawesome-brands-app-store-ios: App Store](https://apps.apple.com/us/app/netnewswire-rss-reader/id1480640210)
```
